components/docs-page.js

```diff
@@ -68,12 +71,12 @@ function PageSections({title, note, heading, sections}) {
       <PageWrapper>
         <Title dangerouslySetInnerHTML={{__html: mdToHTMLUnwrapped(title)}} />
         <Div
-          maxWidth="70rem"
+          maxWidth="35rem"
```

Comment:
I feel that `35rem` is a little too narrow... How about `50rem`? I think that's where we had it at one time...